kimchi/src/circuits/polynomials/foreign_field_add/gadget.rs

```diff
@@ -27,31 +30,40 @@ impl<F: PrimeField + SquareRootField> CircuitGate<F> {
     ///      [n+1]         -> 1 Zero row for bound result
     /// ]
     ///
-    pub fn create(
+    /// INTEGRATION:
+    /// - Wire the range check for result bound manually
+    /// - Connect to public input containing the 1 value for the overflow in the final bound check
+    pub fn create_chain_ffadd(
```

Comment:
Just checking that we still have a way to create additions in a maximally flexible way?  In other words, we still have interfaces where we can perform the "external" checks (i.e. range checks and bound checks) however we like, so we can craft and optimise things around a particular use case...